serverless.yml

```diff
@@ -0,0 +1,33 @@
+
+service: census
+# app and org for use with dashboard.serverless.com
+#app: your-app-name
+#org: your-org-name
+
```

Comment:
can we add serverless-offline to the plugin section?

Reply:
I've updated the files accordingly!